OC4IDS_Database_Data_Import.ipynb

```diff
@@ -6,64 +6,79 @@
         "id": "wlYslPD1uRp2"
```

Comment:
<p>It might be useful here just to have a sentence making it clear that this and the next 2 sections will need repeated for each publisher if multiple publishers need updated</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-contracting/notebooks-oc4ids/pull/39/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Otherwise this all looks fine and works :)

Reply:
If you copy those cells, you'll end up with only the `project-package.json` of the final set of cells. Support for downloading data from multiple publishers at once would need more changes to the notebook so I hope this is OK to merge-as is.

Reply:
ah, okay, sure then it's ok to merge as is then